src/main/java/hw4/Task_1.java

```diff
@@ -0,0 +1,47 @@
+package hw4;
+
+import java.util.Scanner;
+
+public class Task_1 {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        // firstAddWords(scanner);
+        secondAddWords(scanner);
+
+    }
+
+    public static void firstAddWords(Scanner scanner) {
+        int i = 1;
+        String result = " ";
+        while (i < 100) {
```

Comment:
What is if enter 101 word without STOP? So you need while true here instead of < 100